hail/python/test/hail/methods/relatedness/test_identity_by_descent.py

```diff
@@ -45,11 +45,17 @@ def plinkify(ds, min=None, max=None):
     return results
 
 
+def random_dataset():
```

Comment:
I'd suggest making this a fixture:
```python
@pytest.fixture(scope=module)
def ds():
  ...
```

Then each test that wants to use it adds an argument of the same name:
```python
def test_ibd_default_arguments(ds):
  plink_results = plinkify(ds)
  ...
```

The `scope=module` caches the result of the function to avoid recomputing it over the duration of the test module. In this case it doesn't save that much, because it's only caching the construction of the IR, but that's probably slightly better than recomputing it each time.